src/Compilers/CSharp/Portable/Parser/Lexer.cs

```diff
@@ -2293,12 +2293,145 @@ private void LexSyntaxTrivia(bool afterFirstToken, bool isTrailing, ref SyntaxLi
                         {
                             return;
                         }
+
+                    case '>':
```

Comment:
To clarify, the idea is that the skipped text section would be recognized by the scanner.

---
In reply to: [103551073](https://github.com/dotnet/roslyn/pull/15850#discussion_r103551073) [](ancestors = 103551073,103549556)